### PR TITLE
feature/add-adeunis-pulse4

### PR DESCRIPTION
**Pull request by J2thatsme**

### CODE REVIEW

File: types/adeunis/pulse4/default.schema.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
+{
+  "$id": "https://akenza.io/adeunis/pulse4/default.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Default",
+  "type": "object",
+  "properties": {
+    "counterValueT0": {
+      "title": "Counter value time - 0",
```
Comment (mariusgiger) [inline types/adeunis/pulse4/default.schema.json:8]: what does time 0 mean? can you find a better title?

File: types/adeunis/pulse4/status.schema.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+  "$id": "https://akenza.io/adeunis/pulse4/status.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Status",
```
Comment (mariusgiger) [inline types/adeunis/pulse4/status.schema.json:4]: is this lifecycle?

**Review by mariusgiger — APPROVED** [review]

File: types/adeunis/pulse4/default.schema.json (hunk 1 above)
Comment (J2thatsme) [reply types/adeunis/pulse4/default.schema.json]: Most recent measurement.
I renamed it to counterValue. The other keys still have the time indication.

Comment (mariusgiger) [reply types/adeunis/pulse4/default.schema.json]: I mean, how is t measured?

Comment (J2thatsme) [reply types/adeunis/pulse4/default.schema.json]: Well its a variable timeframe when the measurements gets taken.  So f.e if you have 10 measurements in one hour with one datapush. counterValueT1 would be (now - 6m), counterValueT2  (now - 12m) etc. If you take 6 measurements in one hour with one datapush, you would have counterValueT1  (now - 10m), counterValueT2  (now - 20m) etc

**Review by mariusgiger — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,10 +4,16 @@ All notable changes to this project will be documented in this file.
 
 ---
 
-# [1.9.3] - 2021-01-20
+# [1.10.0] - 2021-01-21
```
Comment (mariusgiger) [inline CHANGELOG.md:7]: change the year to 2022

**Review by mariusgiger — APPROVED** [review]

**Pull request merged**